advanced_functionality/multi-container-endpoint/direct-invocation/multi-container-direct-invocation.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>Yes you definitely want to use <code>sagemaker.image_uris.retrieve</code> instead of this string construction. Then you can remove that whole hard-coded region dictionary for account IDs. Something like this:</p><pre class="ql-syntax" spellcheck="false">sagemaker.image_uris.retrieve(framework='tensorflow',region=region,version='2.3.1',image_scope='inference',instance_type='ml.c4.xlarge')
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>